tests/stores/unit/state/ImmutableState.ts

```diff
@@ -214,6 +214,17 @@ describe('state/ImmutableState', () => {
 		});
 	});
 
+	it('should report the length of an array', () => {
+		const state = new ImmutableState();
+		state.apply([{ op: OperationType.ADD, path: new Pointer('foo'), value: [] }]);
+
+		assert.equal(state.path('foo', 'length').value, 0);
```

Comment:
Do you think we should use `state.get` to actually return the value?

Reply:
Fair enough, I've updated this. Though it is a bit funny to replace `path.value` with a call to `get` that's just returning `path.value` anyways. Maybe at some point we should evaluate whether `state.get` will ever have merit a more useful implementation and if not, consider removing it?